From 8ec15c0518b62c6fd11c732697ae7afb44364ad4 Mon Sep 17 00:00:00 2001
From: Michael W <baddaemon87@gmail.com>
Date: Tue, 26 Dec 2017 13:26:08 +0100
Subject: [PATCH 039/249] wiki: Search: Exclude build and installation pages
 from search

* We have build_guides.html and install_guides.html as a common
  adress for these
* Don't spam the search bar when searching for "info" or "build"
* Fix a few page titles so the overview-pages are properly ordered

Change-Id: I0acfbab2e8e9e1f69657374b37c4338c51e232dd
---
 pages/install/ivy.md         | 2 +-
 pages/install/karin.md       | 2 +-
 pages/install/karin_windy.md | 2 +-
 pages/install/suzuran.md     | 2 +-
 search.json                  | 4 +++-
 5 files changed, 7 insertions(+), 5 deletions(-)

diff --git a/pages/install/ivy.md b/pages/install/ivy.md
index f31bcb1..892947a 100644
--- a/pages/install/ivy.md
+++ b/pages/install/ivy.md
@@ -1,6 +1,6 @@
 ---
 sidebar: home_sidebar
-title: Install Lineage on ivy
+title: Install LineageOS on ivy
 folder: info
 # name of the page (/{{permalink}}.html)
 redirect_from: ivy_install.html
diff --git a/pages/install/karin.md b/pages/install/karin.md
index ca7d921..bd3611c 100644
--- a/pages/install/karin.md
+++ b/pages/install/karin.md
@@ -1,6 +1,6 @@
 ---
 sidebar: home_sidebar
-title: Install Lineage on karin
+title: Install LineageOS on karin
 folder: info
 # name of the page (/{{permalink}}.html)
 redirect_from: karin_install.html
diff --git a/pages/install/karin_windy.md b/pages/install/karin_windy.md
index 70b890e..9b70700 100644
--- a/pages/install/karin_windy.md
+++ b/pages/install/karin_windy.md
@@ -1,6 +1,6 @@
 ---
 sidebar: home_sidebar
-title: Install Lineage on karin_windy
+title: Install LineageOS on karin_windy
 folder: info
 # name of the page (/{{permalink}}.html)
 redirect_from: karin_windy_install.html
diff --git a/pages/install/suzuran.md b/pages/install/suzuran.md
index ce0327d..336b21a 100644
--- a/pages/install/suzuran.md
+++ b/pages/install/suzuran.md
@@ -1,6 +1,6 @@
 ---
 sidebar: home_sidebar
-title: Install Lineage on suzuran
+title: Install LineageOS on suzuran
 folder: info
 # name of the page (/{{permalink}}.html)
 redirect_from: suzuran_install.html
diff --git a/search.json b/search.json
index 8f8fb93..0acd69a 100644
--- a/search.json
+++ b/search.json
@@ -8,7 +8,7 @@ search: exclude
 
 {% for page in site.pages %}
 {% unless page.search == "exclude" or page.url contains "_info.html" or page.url contains "_install.html" or page.url contains "_build.html" or page.url contains "devices.html" or page.url contains ".css" %}
-{% if forloop.index0 > 1 %}, {% endif %}
+{% unless page.url contains "/devices" and page.url contains "/build" or page.url contains "/devices" and page.url contains "/install" %}
 {
 "title": "{{ page.title | escape }}",
 "tags": "{{ page.tags }}",
@@ -16,6 +16,8 @@ search: exclude
 "url": "{{ page.url | relative_url }}",
 "summary": "{{page.summary | strip }}"
 }
+{% if forloop.last == false %}, {% endif %}
+{% endunless %}
 {% endunless %}
 {% endfor %}
 
-- 
2.17.1

